packages/smithy-client/src/extensions/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./defaultClientConfiguration";
```

Comment:
should this also export from checksum, or is checksum entirely private to `defaultClientConfiguration`?

Reply:
checksum should be entirely private to defaultClientConfiguration